src/management-system-v2/components/process-export.tsx

```diff
@@ -14,16 +16,51 @@ const exportTypeOptions = [
 
 const exportSubOptions = {
   bpmn: [
-    { label: 'Export with referenced Processes', value: 'imports' },
-    { label: 'Export with Artefacts', value: 'artefacts' },
+    {
+      label: 'with artefacts',
+      value: 'artefacts',
+      tooltip:
+        'Also export html and images used in User-Tasks and images used for other process elements',
+    },
+    {
+      label: 'with referenced processes',
+      value: 'imports',
+      tooltip: 'Also export all referenced processes used in call-activities',
+    },
   ],
   pdf: [
-    { label: 'Export with referenced Processes', value: 'imports' },
-    { label: 'Export with collapsed subprocesses', value: 'subprocesses' },
+    {
+      label: 'with meta-data',
```

Comment:
misspeling; I think valid spelling is either _metadata_ or _meta data_

Reply:
Fixed